app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -167,3 +167,13 @@ Talk to your training provider about the home fee rate, [bursaries and scholarsh
 [Student Finance England](https://www.gov.uk/contact-student-finance-england) has a dedicated service for EU students and for EEA citizens working in the UK.
 
 The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/About-UKCISA) offers advice to international students about what financial support may be available.
+
+## Contact us
+
+If you have general questions about the information on this page, you can email us at teach.inengland@education.gov.uk.
+
+You may be eligible for personalised support with your application to a training provider or school from a [teacher training adviser](/teacher-training-advisers).
```

Comment:
@laurahermionetennant92 I had to read this sentence a few times as I was a bit confused by the training provider or school bit. Perhaps it might be easier to split into 2 sentences? 

You may be eligible for personalised support from a [teacher training adviser](/teacher-training-advisers). They can help with your application to a training provider or school.

Reply:
@laurahermionetennant92 not sure if you updated this bit as I still think it's a bit confusing to read?

Reply:
oh weird, I thought I had